python/amici/ode_export.py

```diff
@@ -3314,6 +3320,7 @@ def cast_to_sym(value: Union[SupportsFloat, sp.Expr, BooleanAtom],
         raise TypeError(f"Couldn't cast {input_name} to sympy.Expr, was "
                         f"{type(value)}")
 
+    # use Pow function with custom derivative
```

Comment:
hmm... that comment looks a bit lost here..

Reply:
indeed!